test/test_ion_text.cpp

```diff
@@ -720,3 +720,426 @@ TEST(IonTextInt, BinaryLiterals) {
     ION_ASSERT_OK(ion_reader_read_int64(reader, &value));
     ASSERT_EQ(-4, value);
 }
+
+/** tests ion_reader_get_value_position for scalar values. */
+TEST(IonTextPosition, PositionOfValues) {
+    const char *ion_text =
+        // Column number cheat sheet:
+        // 1         2         3         4         5         6         7         8         9         10        11        12        13        14
+        // 012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890
+        "  null true 1 2e0 3.0 four 'five' \"six\" 2007-07-07T {{ V2hhdCBhcmUgeW91IGRvaW5nLCBEYXZlPw== }} {{ '''I can't let you do that, Dave.''' }} 42";
```

Comment:
Maybe I'm misunderstanding how to use the cheat sheet, but I was trying to use it to easily look up offsets. The top number is the tens place and the bottom is the ones place. However, there are currently two problems: 1) (minor) the tens place number should start at 0, and 2) it's misaligned by two since the data starts with two spaces, which are included in the offset calculations. Still really helpful though. (The same goes for the cheat sheet for the next test.)

Reply:
Yeah, fixing this.  Also removing the other, less needed one where I check for struct field positions.